### PR TITLE
Update language es_ES

### PR DESCRIPTION
**Pull request by tanrax**
I fixed the translation problem and translated the rest. Could it be mentioned in the table of translators? Thank you

### CODE REVIEW

**Comment by tanrax** [comment]
I fixed the translation problem and translated the rest

**Comment by laurent22** [comment]
Thanks @tanrax. I've already fixed the error I mentioned earlier and for this update it will be part of the next release then (the one after today's release).

To add the name to the translator list, please set the "Last-Translator" key as this what is used to build the list. So it would be something like this:

```
...
"Report-Msgid-Bugs-To: \n"
"Last-Translator: Your Name <youremail@domain.com>\n"
...
```

You may also leave the name as you did in the comment, to keep an history of the changes.

**Comment by tanrax** [comment]
Thanks for the answer! 
I just added it, did I do it right?

**Comment by laurent22** [comment]
Yes looks good 👍

File: CliClient/locales/es_ES.po [hunk 1]
```diff
@@ -463,15 +467,14 @@ msgstr "Descargando recursos..."
...
-#, fuzzy
 msgid ""
 "<tag-command> can be \"add\", \"remove\" or \"list\" to assign or remove "
 "[tag] from [note], or to list the notes associated with [tag]. The command "
 "`tag list` can be used to list all the tags (use -l for long option)."
 msgstr ""
-"<tag-command> puede ser \"add\", \"remove\" o \"list\" para asignar o "
-"eliminar [tag] de [note], o para listar las notas asociadas con [tag]. El "
-"comando `tag list` puede ser usado para listar todas las etiquetas."
+"<tag-command> can be \"add\", \"remove\" or \"list\" to assign or remove "
```
Comment (tessus) [inline CliClient/locales/es_ES.po]: Why did you change the Spanish text to English?

Comment (tanrax) [reply CliClient/locales/es_ES.po]: Oops! That was a mistake.

Comment (tessus) [reply CliClient/locales/es_ES.po]: Thanks for the update.

LGTM.

**Comment by tessus** [comment]
LGTM

**Comment by tessus** [comment]
Thank you for your contribution and PR!
